c_cxx/fns_candy_style_transfer/CMakeLists.txt

```diff
@@ -1,9 +1,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-add_executable(fns_candy_style_transfer "fns_candy_style_transfer.c")
+add_executable(fns_candy_style_transfer "fns_candy_style_transfer.c" "image_file.h")
+if(WIN32)
+  target_sources(fns_candy_style_transfer PRIVATE image_file_wic.cc)
+else()
+  target_sources(fns_candy_style_transfer PRIVATE image_file_libpng.c)
```

Comment:
my understanding is that libpng things (PNG_LIBRARIES, PNG_INCLUDE_DIRS, PNG_LIBDIR) aren't used on windows. perhaps it would be clearer to put them in this block?

Reply:
Yes, You are right.